packages/qdrant-loader/src/qdrant_loader/core/worker/pool.py

```
@@ -0,0 +1,255 @@
+                        ) -> None:
+                            while True:
+                                await asyncio.sleep(current_renewal_interval)
+                                try:
+                                    async with self._queue_io_guard:
+                                        extended = await self._queue.extend_visibility(
+                                            current_job_id,
+                                            current_lease_seconds,
+                                            claim_attempt=current_claim_attempt,
+                                         )
+                                    if not extended:
+                                        lease_lost.set()
```

Ruff (0.15.15)

[warning] Function definition does not bind loop variable `lease_lost`

(B023)